spec/services/unpublish_service_spec.rb

```diff
@@ -47,6 +47,22 @@
       expect(edition.status.details.public_explanation).to eq(public_explanation)
     end
 
+    it "saves the published_status to the withdrawal record" do
+      UnpublishService.new.withdraw(edition, public_explanation, user)
+      edition.reload
+
+      expect(edition.status.details.published_status.state).to eq("published")
```

Comment:
I think this test would be a bit easier to follow if we compared the status before/after, instead of the state. Otherwise, it's a bit surprising to see the literal `published` appearing here.

Reply:
As i'm not testing here whether the status is being set to withdrawn, i'm only checking whether the `published_status` has been set correctly when a withdrawal is created. 

Checking the state is 'published' comes from the factory creation at the top of the page `let(:edition) { create(:edition, :published) }` as the `published_status` is set to the prior status of the edition. However if this isn't clear reading it

I could do:
```
   it "saves the published_status to the withdrawal record" do
      previous_published_status = edition.status
      UnpublishService.new.withdraw(edition, public_explanation, user)
      edition.reload

       expect(edition.status.details.published_status).to eq(previous_published_status)

Reply:
Yeah, I think that would be clearer, as it also matches the description in the 'it' ;-).

Reply:
done